Add how-to docs for catalog sync (#673)
apache/incubator-xtable: website/docs/how-to-catalog-sync.md

current dataset using a bundled jar. You can create this bundled jar by following the instructions
on the [Installation page](/docs/setup). Read through Apache XTable™'s
[GitHub page](https://github.com/apache/incubator-xtable#building-the-project-and-running-tests) for more information.
:::

In this tutorial, we’ll show you how to use Apache XTable™ (Incubating) to enable interoperability between catalogs. 
For example, you can expose a Hudi, Iceberg, or Delta table in Hive Metastore (HMS) and make it available in the AWS Glue Data Catalog—without manually registering each table. 
Additionally, Apache XTable™ (Incubating) allows you to convert the table format metadata. For instance, a Delta table in HMS can be exposed as an Iceberg table in Glue.

## Pre-requisites
1. Source table(s) (Hudi/Delta/Iceberg) already written to your local storage or external storage locations like S3/GCS/ADLS. 
   If you don't have the source table written in place already, you can follow the steps in this [tutorial](/docs/how-to#create-dataset) to set it up.
2. Clone the Apache XTable™ (Incubating) [repository](https://github.com/apache/incubator-xtable) and create the

>    `xtable-utilities_2.12-0.2.0-SNAPSHOT-bundled.jar` by following the steps on the [Installation page](/docs/setup)

Reviewer: Will this be 0.3.0 going forward?

Author: Yes after the release is published, docs will be updated.

:::

:::note Note:
Authentication for AWS is done with `com.amazonaws.auth.DefaultAWSCredentialsProviderChain`.
To override this setting, specify a different implementation with the `--awsCredentialsProvider` option.
:::

In your terminal under the cloned Apache XTable™ (Incubating) directory, run the below command.

```shell md title="shell"
java -cp xtable-utilities/target/xtable-utilities_2.12-0.2.0-SNAPSHOT-bundled.jar org.apache.xtable.utilities.RunCatalogSync --catalogSyncConfig my_config_catalog.yaml
```

**Optional:**
Now, if you check your target catalog, you'll see the external tables have been created and are ready to be queried.

## Conclusion
In this tutorial, we explored how to set up Apache XTable™ (Incubating) to automatically read tables from a source catalog and create interoperable external tables in target catalogs.

> Once synced, these tables are immediately queryable from the target catalog—no manual registration needed.

Reviewer: ```suggestion
Once synced, these tables are immediately queryable from the target catalog without manual registration.
```
